logging/google/cloud/logging/handlers/_helpers.py

```diff
@@ -36,4 +38,44 @@ def format_stackdriver_json(record, message):
         'severity': record.levelname,
     }
 
+    # Make a best effort to add the trace id.
+    # First try to extract the trace_id from HTTP header, only support flask framework for now.
+    trace_id = parse_flask_request_header()
```

Comment:
@duggelz The issue is not **THAT** we provide purpose-built helpers, just **WHERE / HOW**. (Hence my suggestion for a better fix.)